Common/src/Storage/TaskDataFields.cs

```diff
@@ -0,0 +1,52 @@
+// This file is part of the ArmoniK project
+// 
+// Copyright (C) ANEO, 2021-2023. All rights reserved.
+// 
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+// 
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace ArmoniK.Core.Common.Storage;
+
+/// <summary>
+///   Fields available in <see cref="TaskDataHolder" />
+/// </summary>
+public enum TaskDataFields
```

Comment:
I don't see the point of adding xml-doc for each member of the Enum. @ngruelaneo @lemaitre-aneo, your thoughts ?

Reply:
Yes!